java/code/src/com/redhat/rhn/taskomatic/task/RhnJavaJob.java

```diff
@@ -84,8 +82,12 @@ protected void executeExtCmd(String[] args)
             }
             throw new JobExecutionException(
                     "Command '" + Arrays.asList(args) +
-                    "' exited with error code " + exitCode +
-                    (msg.isBlank() ? "" : ": " + msg));
+                            "' exited with error code " + exitCode +
+                            (msg.isBlank() ? "" : ": " + msg));
         }
     }
+
+    protected void executeExtCmd(String[] args) throws JobExecutionException {
+        executeExtCmd(args, false);
```

Comment:
If by default you pass it as false, I think it will not store the stdout.

Reply:
The parameter here is `noStdoutLog`, the logic is inverted in that function